src/pid4cat_model/schema/pid4cat_model.yaml

```diff
@@ -60,6 +60,7 @@ classes:
   PID4CatRelation:
     description: >-
       A relation between PID4CatRecords or between a PID4CatRecord and other resources with a PID.
+      # TODO: please provide and example for this class 
```

Comment:
Please remove. General todos should better be issues. There is already #24 for this.